samples/Azure.Management.Storage/Generated/Extensions/SubscriptionExtensions.cs

```diff
@@ -32,19 +32,19 @@ private static StorageAccountsRestOperations GetStorageAccountsRestOperations(Cl
         /// <param name="subscription"> The <see cref="SubscriptionOperations" /> instance the method will execute against. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
         /// <returns> A collection of resource operations that may take multiple service requests to iterate over. </returns>
-        public static AsyncPageable<StorageAccount> ListStorageAccountsAsync(this SubscriptionOperations subscription, CancellationToken cancellationToken = default)
+        public static AsyncPageable<StorageAccount> GetStorageAccountsAsync(this SubscriptionOperations subscription, CancellationToken cancellationToken = default)
```

Comment:
Should we call this `GetAllStorageAccountsAsync` to align other changes?

Reply:
this one is following
List[Noun]s -> Get[Noun]s